deno/payloads/v10/channel.ts

```diff
@@ -223,6 +225,8 @@ export interface APIThreadChannel
 	last_message_id?: Snowflake | null;
 }
 
+export type APIGuildForumChannel = APIGuildTextChannel<ChannelType.GuildForum>;
```

Comment:
Should we name them APIForumChannel instead? (considering APINewsChannel)... I'd say no, but I want thoughts

Reply:
The type is documented as `GUILD_FORUM` so if we wanna stay in line with API naming conventions it should be `APIGuildForumChannel` imo.

Reply:
Bleh, we should probably rename news channels sometime then...